server/container/guice/memory-guice/src/main/java/org/apache/james/MemoryJamesServerMain.java

```diff
@@ -115,6 +116,7 @@ public static void main(String[] args) throws Exception {
             .build();
 
         GuiceJamesServer.forConfiguration(configuration)
+            .combineWith(new LuceneSearchMailboxModule())
```

Comment:
No! It's inMemory not lucene

You by the way are lying to your command line users by reindeing docs in a non used searched index

This is madness.

If you really want a CLI, please bind a ThrowingSearchIndex instead - everybody gonna be healthier from this choice.

Note : maybe CLI and thus JMX is not needed at all. Thoughts @mbaechler ?